source/StartConditions.cpp

```diff
@@ -44,7 +49,7 @@ void StartConditions::Load(const DataNode &node)
 	// When a plugin modifies an existing starting condition, default to
 	// clearing the previously-defined description text. The plugin may
 	// amend it by using "add description"
-	bool clearDescription = !description.empty();
+	bool clearDescription = !infoByState[StartState::UNLOCKED].description.empty();
```

Comment:
shouldnt this happen for all states? either independently or all getting cleared together?
this only happens with the unlocked state for now?

Reply:
Look at the loadstate function

Reply:
Looks good there but shouldnt we use that method for unlocked as well? Seems like some duplicate code but mb I'm missing smth

Reply:
unlocked is in the same line as other nodes, so i cant really use it for unlocked

Reply:
I've created a LoadStateChild function that deduplicates the code between Load and LoadState.